integrations/github/src/haystack_integrations/prompts/github/context.py

```diff
@@ -0,0 +1,175 @@
+haystack_context_prompt = """
+
+Haystack-Agent was specifically designed to help developers with the Haystack-framework and any Haystack related
```

Comment:
Remove extra new lines
```suggestion
haystack_context_prompt = """Haystack-Agent was specifically designed to help developers with the Haystack-framework and any Haystack related
```

Reply:
Done.